R/importNGWMN_wml2.R

```diff
@@ -32,7 +32,7 @@
 #' 
 #' 
 #TODO: separate id and agency name, give also as separate dimensions
-importNGWMN_wml2 <- function(input, asDateTime=FALSE, tz=""){
+importNGWMN_wml2 <- function(input, asDateTime=FALSE, tz){
```

Comment:
Any reason not to set the default to "UTC" like the rest of dataRetrivial? I find it confusing when there's a default argument in the middle of a set of arguments, and one that doesn't have a default. Then, I can't just take advantage of that default by saying:
```
importNGWMN(input, "UTC")
```

So, either move `asDateTime` to the right, or add a default of "UTC" to the `tz` (my vote is the 2nd).